src/Generator.Bind/Program.cs

```diff
@@ -62,6 +63,8 @@ public static class Program
         /// <returns>An integer, indicating success or failure. On a failure, a nonzero value is returned.</returns>
         private static async Task<int> Main(string[] args)
         {
+            // force the GC off for a bit more speed.
+            GCSettings.LatencyMode = GCLatencyMode.Batch;
```

Comment:
How much of a difference does this make? I'd prefer to avoid arcane code if the benefits are minimal :P

Reply:
It's the correct GC mode for a batch tool. The default is designed for low-latency client-facing operations.

Reply:
Think of the numbers, mason.

(excuse the Black Ops 2 reference, but can we have some stats?)

Reply:
I swapped based on the fact that it's the correct mode for the type of processing we're doing.

But here are your numbers for interest sake:
Default time in GC: 764ms
Batch time in GC: 693ms

Default total time:  10.4340316
Batch total time: 10.3568002

Time in GC was profiled using dotTrace (timeline mode).
Total time was read from the app output with no profiling attached.

Reply:
Nice!